common/model/db/tables/record/table.js

```diff
@@ -25,3 +31,29 @@ export default class TableRecord extends TableSurvey {
 }
 
 TableRecord.columnSet = columnSet
+
+/**
+ * Transforms the result row of a query on the record table into a Record object.
+ *
+ * @param {!number} surveyId - The ID of the survey.
+ * @param {boolean} [includeValidationFields=true] - Whether to include all the Validation fields.
+ * @returns {Record} - The record object.
+ */
+TableRecord.dbTransformCallback = (surveyId, includeValidationFields = true) => (record) => {
```

Comment:
call this method simply transformCallback and add it to a separate file.
then TableRecord.transformCallback = transformCallback

Reply:
ok